projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/components/query-check-your-answers/query-check-your-answers.component.ts

```diff
@@ -14,10 +14,14 @@ export class QueryCheckYourAnswersComponent implements OnInit {
 
   @Input() public formGroup: FormGroup;
   @Input() public queryItem: QueryListItem;
+  @Input() public context: string;
```

Comment:
Can you please make it so context's type is QueryItemType?

Reply:
Also optionally... I would change either 'context' or QueryItemType so they match. Maybe QueryCreateContext?